src/Controller/SearchController.php

```diff
@@ -85,6 +85,16 @@ public function __invoke(Request $request)
             $request->query->set('locale', $channel->getDefaultLocale()->getCode());
         }
 
+        if (!$request->query->has('sort')) {
+            if (null !== $request->get('taxonCode')) {
```

Comment:
`$request->has('taxonCode')`?

Also, why don't we narrowing the parameters scope to query parameters only?